### PR TITLE
Update Mazer docs

### PR DESCRIPTION
**Pull request by alikins**
Work in progress update of mazer docs

### CODE REVIEW

File: docs/docsite/rst/mazer/examples.rst [hunk 1]
```diff
@@ -10,468 +10,346 @@ Examples
...
-
-A version number can also be passed using the same ',' separated format. For example:
-
-.. code-block:: bash
-
-    $ mazer install git+https://github.com/geerlingguy/ansible-role-nginx,2.6.0
+Collections are a new way to package and distribute ansible related content.
+See :ref:`creating_collections` for examples.
 
+Collections are available starting with version 3.2 of the Galaxy server and version 2.9 of ansible.
```
Comment (ironfroggy) [inline docs/docsite/rst/mazer/examples.rst]: 2.9 should be 2.8, even though its "tech preview" we do say 2.8 elsewhere and 2.9 won't be available yet.

File: docs/docsite/rst/mazer/examples.rst [hunk 2]
```diff
@@ -10,468 +10,346 @@ Examples
...
-Installing Repositories With Multiple Roles
--------------------------------------------
-
-Starting with version 3.0 of the Galaxy server, many roles can be combined into a single git repository. Prior to this role repositories
-were structured to only contain a single role. Mazer is able extract all of the roles from the repostory, and install them to the
-local file system.
-
-To install a multi-role repository from Galaxy, pass the *namespace.repository_name* to the install command. The following
-installs the `testing.ansible-testing-content repository <https://galaxy.ansible.com/testing/ansible-testing-content>`_ from
+To install a collection from Galaxy, pass the *namespace.repository_name* to the install command. The following
```
Comment (ironfroggy) [inline docs/docsite/rst/mazer/examples.rst]: not "repository_name"

File: docs/docsite/rst/mazer/index.rst [hunk 3]
```diff
@@ -9,21 +9,20 @@ Mazer is a new command-line tool for managing `Ansible <https://github.com/ansib
...
     production environments, nor is it currently intended to replace the `ansible-galaxy` command-line tool.
 
     If you're installing Ansible content in a production environment, or need assistance with Ansible, please visit the `Ansible Project <https://github.com/ansible/ansible>`__,
     or the `Ansible docs site <https://docs.ansible.com>`_.
 
 .. note::
-    
-    Before installing roles with Mazer, review :ref:`using_mazer_content`. Mazer installs content different from
+
+    Before installing roles with Mazer, review :ref:`using_collections_in_playbooks`. Mazer installs content different from
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/index.rst]: Mazer no longer install Roles. This should be re-worded, me thinks.

File: docs/docsite/rst/mazer/index.rst [hunk 4]
```diff
@@ -9,21 +9,20 @@ Mazer is a new command-line tool for managing `Ansible <https://github.com/ansib
...
     the way ``ansible-galaxy`` does.
 
-    Mazer is most useful when used with a version of Ansible that understands mazer installed content.
-    Currently that means the `'mazer_role_loader' branch of ansible <https://github.com/ansible/ansible/tree/mazer_role_loader>`_.
-    See :ref:`installing_the_companion_branch_of_ansible` for details.
+    Mazer is most useful when used with a version of Ansible that understands *collections*.
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/index.rst]: Simplify the wording. i.e., "Mazer requires Ansible 2.8 or higher"

File: docs/docsite/rst/mazer/configure.rst [hunk 5]
```diff
@@ -22,14 +22,8 @@ an example configuration file:
...
-    content_path: ~/.ansible/content
-    options:
-      local_tmp: ~/.ansible/tmp
-      role_skeleton_ignore:
-         - ^.git$
-         - ^.*/.git_keep$
-      role_skeleton_path: null
-      verbosity: 0
+    collections_path: ~/.ansible/collections
+    global_collections_path: /usr/share/ansible/collections
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/configure.rst]: What about the API key?

File: docs/docsite/rst/mazer/index.rst (hunk 3 above)
Comment (alikins) [reply docs/docsite/rst/mazer/index.rst]: updated

File: docs/docsite/rst/mazer/index.rst (hunk 4 above)
Comment (alikins) [reply docs/docsite/rst/mazer/index.rst]: updated

File: docs/docsite/rst/mazer/configure.rst (hunk 5 above)
Comment (alikins) [reply docs/docsite/rst/mazer/configure.rst]: Added api_key to the examples and some help blurbs based on mazer readme

File: docs/docsite/rst/mazer/examples.rst (hunk 2 above)
Comment (alikins) [reply docs/docsite/rst/mazer/examples.rst]: updated
